lib/ok.ex

```diff
@@ -161,22 +161,39 @@ defmodule OK do
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
       ...>   b <- safe_div(a, 2)
+      ...>   OK.success a + b
+      ...> end
+      {:ok, 6.0}
+
+      iex> OK.with do
+      ...>   a <- safe_div(8, 2)
+      ...>   b <- safe_div(a, 2)
       ...>   {:ok, a + b}
       ...> end
       {:ok, 6.0}
 
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
+      ...>   b = 2.0
+      ...>   OK.success a + b
```

Comment:
Hrmm, I come from a non-Ruby background and I am used to _always_ using parens. That was actually probably the _most_ confusing thing for me: reading example code that left off the parens, especially when combined with the optional square brackets in a final `opts` argument keyword list. Very confusing. :confused:

That said, I personally _do_ like the lack of parens in limited cases, and `OK.success` is one of them (my only others are when using `IO.puts` and `Logger.debug/warn/error`). There is no compiler warning and it compiles correctly (tests pass). 

Do you have a link to guidance specifically about this or are you thinking more generally?